opencontrail/control.sls

```diff
@@ -91,4 +91,19 @@ opencontrail_control_entrypoint:
 
 {%- endif %}
 
-{%- endif %}
\ No newline at end of file
+{%- endif %}
+
+{%- if common.vendor == "juniper" %}
```

Comment:
Ifmap is not related to control, but config role! Please remove this from control.sls.